stripper/ze_aaa_oh_final_final_final.cfg

```diff
@@ -0,0 +1,18 @@
+;oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh oh
+;implement various vscript fixes via aaa_oh_fix.nut
+modify:
+{
+	match:
+	{
+		"classname" "logic_script"
+		"targetname" "chinny_script"
+	}
+	delete:
+	{
+		"vscripts" "chinny/musicloop.nut chinny/chinny_general.nut chinny/chinny_toledo.nut chinny/biggest_bird.nut chinny/chinny_cincinnati.nut  chinny/warmup.nut"
+	}
+	insert:
+	{
+		"vscripts" "chinny/musicloop.nut chinny/chinny_general.nut chinny/chinny_toledo.nut chinny/biggest_bird.nut chinny/chinny_cincinnati.nut chinny/warmup.nut aaa_oh_fix.nut"
+	}

```

Comment:
VScripts in the GitHub need to be referred to as `gfl/filename.nut`, see [readme](https://github.com/gflclan-cs-go-ze/ZE-Configs#vscripts). You can also just use a `replace:` block here, it does delete/insert in one.

Reply:
Really should have read the readme, my bad.

I am going to assume the vscript itself is in the correct directory.